### ISSUE TITLE
mpv window won't till 

### ISSUE THREAD

**Issue by imagnma** [koekeishiya/chunkwm#478]
Hello, 

every time I run command `[mpv] ---.mp4` the new window pop up and won't tile 

i tried this `chunkc tiling::rule --owner mpv --state tile` and still

anyhelp . thanks

**Comment by imagnma** [comment]
tried after opening it alt - t : chunkc tiling::window --toggle float;\ and still it just wont work on it 

by the wey it is installed by pip3 i think

**Comment by kylef** [comment]
I can confirm this behaviour, it seems mpv window is not tracked by chunkwm. The output of `chunkc tiling::query --desktop windows` does not include mpv.

If I focus from iTerm to mpv you can see that it doesn't activate mpv:

```
2018-08-03 01:01:48 DEBUG: chunkwm: processing event of type 'ChunkWM_ApplicationActivated'
2018-08-03 01:01:48 DEBUG: chunkwm: processing event of type 'ChunkWM_ApplicationDeactivated'
2018-08-03 01:01:48 DEBUG: 98049:iTerm2 deactivated
```

Going from mpv to iTerm:

```
2018-08-03 01:01:50 DEBUG: chunkwm: processing event of type 'ChunkWM_ApplicationActivated'
2018-08-03 01:01:50 DEBUG: 98049:iTerm2 activated
2018-08-03 01:01:50 DEBUG: chunkwm:Tiling:focused_window_float
2018-08-03 01:01:50 DEBUG: chunkwm: processing event of type 'ChunkWM_ApplicationDeactivated'
2018-08-03 01:01:50 DEBUG: chunkwm: processing event of type 'ChunkWM_PluginBroadcast'
```

**Comment by koekeishiya** [comment]
See https://github.com/koekeishiya/chunkwm/issues/108

TL;DR

When MPV is launched via the terminal, it spawns as a daemon (background) process, and is therefore ignored by chunkwm. 

There is no way to fix this, because we have no way of determining if this process really IS a daemon process or not, except for this flag that appears to be incorrectly from time to time.

I solved this by making an *mpv* alias: `alias mpv='open -na /Applications/mpv.app'`
